cli-plugins/manager/manager.go

```diff
@@ -42,8 +43,9 @@ type notFound interface{ NotFound() }
 
 // IsNotFound is true if the given error is due to a plugin not being found.
 func IsNotFound(err error) bool {
-	if e, ok := err.(*pluginError); ok {
-		err = e.Cause()
+	var e *pluginError
+	if errors.As(err, &e) {
+		err = e.Unwrap()
```

Comment:
Would you be able to add more context to the commit on why this needs to change? 
The error could also be wrapped multiple times before it's a `notFound` type. Are we sure we never wrap the error more than once?